.pre-commit-config.yaml

```diff
@@ -10,35 +10,8 @@ repos:
     -   id: check-yaml
     -   id: debug-statements
     -   id: end-of-file-fixer
-        exclude: "resources/.*|docs/make.bat"
     -   id: trailing-whitespace
     -   id: mixed-line-ending
         args: ['--fix=lf']
-        exclude: "docs/make.bat"
```

Comment:
I'm uncertain about this. Bat-files are for windows users and they use different line endings, but there is also git autocrlf: https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings#global-settings-for-line-endings

However, we can just do it and see if anybody mentions issues.